src/tiledbsoma_ml/_csr.py

```diff
@@ -2,6 +2,7 @@
 # Copyright (c) 2021-2024 TileDB, Inc.
 #
 # Licensed under the MIT License.
+"""CSR sparse matrix implementation, optimized for incrementally building from COO matrices."""
```

Comment:
this entire class is private (and will be swapped out at some point), so it should probably be noted as "private" either in the module-level comment, or by underscore-prefixing the class name.